airbyte-workers/src/main/java/io/airbyte/workers/config/DatabaseBeanFactory.java

```diff
@@ -92,7 +92,7 @@ public Flyway jobsFlyway(@Named("jobs") final FlywayConfigurationProperties jobs
   @Requires(env = WorkerMode.CONTROL_PLANE)
   public ConfigPersistence configPersistence(@Named("configDatabase") final Database configDatabase,
                                              final JsonSecretsProcessor jsonSecretsProcessor) {
-    return DatabaseConfigPersistence.createWithValidation(configDatabase, jsonSecretsProcessor);
```

Comment:
We may also be able to remove the creation of the `JsonSecretsProcessor` bean from the `ApplicationBeanFactory` if in fact it is no longer used by any other beans as a result of this change.